lib/linguist/languages.yml

```diff
@@ -2824,6 +2824,20 @@ Kaitai Struct:
   - ".ksy"
   tm_scope: source.yaml
   language_id: 818804755
+KeyValues:
```

Comment:
I think we might need a more specific name than this, maybe "Valve Data Format"?

My reasoning is "KeyValues" is very generic and I know many examples where people will think this means key value stores which definitely isn't the case.